pyproject.toml

```diff
@@ -31,20 +30,30 @@ packages = [
 Repository = 'https://github.com/humanloop/humanloop-python'
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
```

Comment:
Why is this being bumped? That could negatively impact existing users.

Reply:
The instrumentors require Python>=3.9

Reply:
Please check with fern that this won't break anything. Also, you didn't acknowledge the point about it impacting existing users.

Reply:
I will check with Fern and I am looking for a workaround

Reply:
Per today standup, we decided to keep the bump to 3.9 and check with customers if it's a breaking change